atom/browser/api/atom_api_auto_updater.cc

```diff
@@ -105,8 +105,12 @@ void AutoUpdater::OnWindowAllClosed() {
   QuitAndInstall();
 }
 
-void AutoUpdater::SetFeedURL(mate::Arguments* args) {
-  auto_updater::AutoUpdater::SetFeedURL(args);
+void AutoUpdater::SetFeedURL(const std::string feedURL) {
```

Comment:
```suggestion
void AutoUpdater::SetFeedURL(const std::string& feedURL) {
```

Can this be a reference instead of copying?